frontend/ui/data/css_tweaks.lua

```diff
@@ -191,40 +191,72 @@ h1, h2, h3, h4, h5, h6 { hyphens: none !important; }
     {
         title = _("Miscellaneous"),
         {
-            id = "table_row_odd_even";
-            title = _("Alternate background color of table rows"),
-            css = [[
+            title = _("Tables"),
+            {
+                id = "table_full_width";
+                title = _("Make tables use the full width"),
+                description = _("Have table use the full width of the page. (Tables with small content now uses only the width needed to display that content. This restores the previous behaviour.)"),
+                css = [[table { width: 100% !important; }]],
+            },
+            {
+                id = "table_td_width_auto";
+                title = _("Ignore table and cells specified widths"),
```

Comment:
```suggestion
                title = _("Ignore specified table and cell widths"),
```

author … widths?
document … widths?
publisher … widths?